writer/proto/resource.go

```diff
@@ -96,6 +96,18 @@ func GeneratedResourceMessage(r *parser.ParsedResource) (*builder.MessageBuilder
 		switch p.Type {
 		case schema.Type_STRING:
 			typ = builder.FieldTypeString()
+		case schema.Type_INT32:
+			typ = builder.FieldTypeInt32()
+		case schema.Type_INT64:
+			typ = builder.FieldTypeInt64()
+		case schema.Type_BOOLEAN:
+			typ = builder.FieldTypeBool()
+		case schema.Type_DOUBLE:
+			typ = builder.FieldTypeDouble()
+		case schema.Type_FLOAT:
+			typ = builder.FieldTypeFloat()
+		default:
+			return nil, fmt.Errorf("%s does not have type support in protobuf", p.Type)
```

Comment:
maybe choose an error message that also implies we might just not have that mapping encoded? I don't think we should say that "it doesn't have support" - this isn't exhaustive yet.

Something like "mapping for type %s not found"